openforcefield/tests/test_molecule.py

```diff
@@ -2764,6 +2764,43 @@ def test_get_bond_between(self):
         with pytest.raises(NotBondedError):
             mol.get_bond_between(hydrogens[0], hydrogens[1])
 
+    @pytest.mark.parametrize(
+        ("smiles", "n_rings"),
+        [
+            ("CCO", 0),
+            ("c1ccccc1", 1),
+            ("C1CC2CCC1C2", 2),  # This should probably be 3?
```

Comment:
I agree strange that RDKit does not pick this up

Reply:
I think some of the discussion in this thread can be summarizes as follows:

* This method is likely to find all rings for most structures, but will miss some rings when multi-cyclic moieties like in norbornane are present
* This method guarantees (?) that a bond or atom will know _if_ it is in a ring, but possibly not _all_ rings it is in. 

Does this work for your use cases, for now?